vectorize-pixelart: add page (#5346)
tldr-pages/tldr: pages/common/vectorize-pixelart.md

# vectorize-pixelart

> > Convert raster pixel art graphics to vector.

Reviewer: ```suggestion
> Convert PNG pixel art graphics to SVG/EPS vector images.
```

> > More information: <https://github.com/und3f/vectorize-pixelart>

Reviewer: ```suggestion
> More information: <https://github.com/und3f/vectorize-pixelart>.
```

- Vectorize a PNG image to an SVG vector:

`vectorize-pixelart {{image.png}} {{output.svg}}`

> - Vectorize to an EPS vector:

Reviewer: ```suggestion
- Convert a PNG image to an EPS image:
```

> - Vectorize a PNG image to an SVG vector:
> 
> `vectorize-pixelart {{image.png}} {{output.svg}}`
> 
> - Vectorize to an EPS vector:
> 
> `vectorize-pixelart {{image.png}} {{output.eps}}`

Reviewer 2: To be fair, as both of these do the same thing (just with different output formats) and don't have any flags, we can probably do a single example with something like the following:
```suggestion
- Convert a PNG image to a vector image format:

`vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|eps}}`
```

Reviewer: What about `{{path/to/output.svg|.eps}}` (with `|.eps`)?

Reviewer 2: I'm not entirely sure what our standard is for multiple extensions.

# vectorize-pixelart

> Convert PNG pixel art graphics to SVG/EPS vector images.
> More information: <https://github.com/und3f/vectorize-pixelart>.

- Convert a PNG image to a vector image format:

> `vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|eps}}`

Reviewer: Let's just do this for now.  Someone can come a long and discuss this for the long run later.
```suggestion
`vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|.eps}}`
```
